collector/__init__.py

```diff
@@ -244,7 +245,9 @@ class Collector(SchedulerComponent):
     _MAX_NIGHT_EVENT_TIME: ClassVar[Time] = Time('2100-01-01 00:00:00', format='iso', scale='utc')
 
     # The number of milliarcsecs in a degree, for proper motion calculation.
-    _MILLIARCSECS_PER_DEGREE: ClassVar[int] = 1296000000
+    _MILLIARCSECS_PER_DEGREE: ClassVar[int] = 60*60*1000
+
+    _EPOCH2TIME: ClassVar[Mapping[float, Time]] = {2000.0: Time('2000-01-01 00:00:00', format='iso', scale='utc'), }
```

Comment:
What are we going to do in the case of other epochs? There should be a place where, if we encounter another epoch value, we add it to this, no? I would think we could just add a line as per below.

Reply:
Incidentally, I'm not sure that this is right. If you look up J2000:

"January 1, 2000
The currently-used standard epoch "J2000" is defined by international agreement to be equivalent to: The Gregorian date January 1, 2000, at 12:00 TT (Terrestrial Time). The Julian date 2451545.0 TT (Terrestrial Time)."